### ISSUE TITLE
[Object] - Black Lotus (various issues)

### ISSUE THREAD

**Issue by Ashfeenix** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#91]
**Issue/s:**

When looted, only one Black Lotus should be available, rather than 1-3. In addition, only one Black Lotus object should be spawned at a time (per zone).

**Proof/Information:**
http://wowwiki.wikia.com/wiki/Black_Lotus - _Only one Black Lotus will spawn at a time in the zone._

Proof that only one lotus should be looted per object - "Pending"

**Replication:**

.go object "Black Lotus"
.learn 8390 (find herb)

Look around, it is very likely you will see more than one lotus node in the area, sometimes even two in the same spot.

**Comment by higuysitme** [comment]
There is no point of fixing it. We have tbc amount of herbs all around, its not a problem of blacklotus. We should have less dreamfoil, gromsblood, etc.  Looks kinda shady to me to be honest. Most likely you someone who stocked thousands of bl. What the point doing it for real? Do you really want chinese camp every spawn? Do you want cash out or something? Dont give that "Its not blizzlike" crap answer. You can call blizzlike only raiding.

**Comment by Ifixstuffk** [comment]
Markets flooded, deserves a fix.

**Comment by higuysitme** [comment]
Yes you right,  people like me and you have hundreds of free black lotus. Lets fix it right now and make people struggle when bwl just got realeased :D What kind of logic is that I dont get it.

**Comment by Ifixstuffk** [comment]
He's a GM. Not a lotus stockpiler. This isn't a discussion, we are trying to fix bugs.

**Comment by higuysitme** [comment]
That does not change the fact that we have too many herb spawns. If you gonna do something you have to rewrite every single herb and make it vanilla amount of spawns. Right now we have twice amount of herb spawns then it should to be. Switch in top right corner from vanilla to tbc on wow-one database.
It supposed to be 18 spawn points of dreamfoil in Azshara. https://www.wow-one.com/database/index.php?object=176584 
But we have 105 right now. Same goes with black lotus. If going to fix it then do it entirely.

**Comment by Ifixstuffk** [comment]
true

**Comment by dagochen** [comment]
How you know it should be only 18?
